Vagrantfile

```diff
@@ -51,6 +68,30 @@ def configureMachineSize(vmCfg, vcpuCount, memSize)
 	return vmCfg
 end
 
+def configureUbuntuNomadProvisioners(vmCfg, hostname, ip)
+    vmCfg.vm.provision "shell",
+    		path: './dev/vagrant/scripts/vagrant-ubuntu-deps.sh',
+    		privileged: true
```

Comment:
Mixed tabs and spaces, or too many spaces. Possibly.

Reply:
Fixed